Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
srun: error: enc1-node2: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc1-node6: task 0: Exited with exit code 1
srun: error: enc1-node6: task 0: Exited with exit code 1
srun: error: enc1-node6: task 0: Exited with exit code 1
srun: error: enc1-node6: task 0: Exited with exit code 1
srun: error: enc1-node6: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
srun: error: enc1-node8: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
srun: error: enc1-node11: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc1-node7: task 0: Exited with exit code 1
srun: error: enc1-node7: task 0: Exited with exit code 1
srun: error: enc1-node7: task 0: Exited with exit code 1
srun: error: enc1-node7: task 0: Exited with exit code 1
srun: error: enc1-node7: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node2: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc1-node5: task 0: Exited with exit code 1
srun: error: enc1-node5: task 0: Exited with exit code 1
srun: error: enc1-node5: task 0: Exited with exit code 1
srun: error: enc1-node5: task 0: Exited with exit code 1
srun: error: enc1-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc0-node5: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
srun: error: enc1-node9: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
srun: error: enc0-node6: task 0: Exited with exit code 1
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
srun: error: enc1-node10: task 0: Exited with exit code 1
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
srun: error: enc1-node4: task 0: Exited with exit code 1
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
  File "clusterSweep.py", line 15, in <module>
Traceback (most recent call last):
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
  File "clusterSweep.py", line 15, in <module>
NameError: name 'a_stdp' is not defined
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
Traceback (most recent call last):
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
Traceback (most recent call last):
  File "clusterSweep.py", line 15, in <module>
    print(t_sr,t_stdp,t_stdp_asymm,a_stdp,f,k,fr)
NameError: name 'a_stdp' is not defined
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node4: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc0-node3: task 0: Exited with exit code 1
srun: error: enc0-node7: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node3: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc1-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
srun: error: enc0-node1: task 0: Exited with exit code 1
